compute/compute.go

```diff
@@ -2,29 +2,27 @@ package compute
 
 import (
 	"encoding/xml"
-	"reflect"
 	"fmt"
-"github.com/megamsys/opennebula-go/api"
+
+	"github.com/megamsys/opennebula-go/api"
 	"github.com/megamsys/opennebula-go/flavor"
-"github.com/megamsys/opennebula-go/xmlUtil"
+	"github.com/megamsys/opennebula-go/xmlUtil"
 )
 
 type VirtualMachine struct {
-	OpenNebulaTemplateName   string
+	OpenNebulaTemplateName string
```

Comment:
Why do you need the prefix `OpenNebula`
I see fields like `Bootstrap`, `RunList`, `Distro`, `SSHUser`, `SSHPort` will not be of any use